config/onprem-iso-config.ign

```diff
@@ -21,15 +21,15 @@
         "path": "/etc/assisted-service/create_certs.sh",
         "contents": {
           "compression": "gzip",
-          "source": "data:;base64,H4sIAAAAAAAC/5zPsW6DMBAG4J2nuNL5cKCAklQMiClL2qp9AWNfExdjUt+lhbevImXqEinbDfd/v/7HB9W7oHrNx2QcrIuAJ1AkRmlmx0IWmeKPM6TCwYUZDUXhZDpRYPYQ6RtwrlYbwDBZYkCrF4anugLkc/8FqeqarlXvH81bp16a9orCLrBo7ymqbt/kmyLL63WWF0VWpYDaWpoF0otARlovez1Ss3vd/v8M9DvQApH1tliVa8CBlukstxdcbyb/ye4QyGYXB+/MmijPiTmOk4W6LO9sT/4CAAD//9U0UEGMAQAA"
+          "source": "data:;base64,H4sIAAAAAAAC/5yPsU7DMBCG9zzFEeaLK0grKMoQhYUlCJUXcOxrY+LYwXeF5O1RUdW1Urd/+b5P//2d6lxQneY+GwfrEuAEisQozexYyCJT+nGGVDi4MKOhJJzFiQKzh0TfgPN69QwYoiUGtHpheNysAfnYfUGumqqp1e6z+mjUe1WfpfAWWLT3lFTTVpeUnlzho9G++A/0bi+FizmgtpZmgfzkJCO1l1aPVL22u+11ONDvQAsk1tuHVfkEONASj3L95nkz+T27QyBbnDx4I2uSvGSmH6OFTVneWM/+AgAA//9PXnJesQEAAA=="
```

Comment:
All of these scripts should be able to be reviewed as well. Is there some better way to store these so that I don't have to unpack all of this every time new changes come in?

Maybe we should post the contents or the patch in a comment if we really have to keep these in this form?
@mhrivnak any ideas?

Reply:
The .ign file is actually generated from fcc.yaml, which has the readable content.

Reply:
Lol just realised this is generated and the scripts are below. :facepalm: